addons/pos_sms/models/pos_order.py

```diff
@@ -0,0 +1,20 @@
+from odoo import models
+
+
+class PosOrder(models.Model):
+    _inherit = 'pos.order'
+
+    def action_sent_message_on_sms(self, name, partner, ticket_image):
```

Comment:
it would be easier to just have a `phone_number` argument instead of a `partner` object